pwiz_tools/Skyline/TestTutorial/CustomReportsTutorialTest.cs

```diff
@@ -494,14 +495,23 @@ protected bool DoQualityControlSummaryReports()
             RunUI(SkylineWindow.ShowPeakAreaReplicateComparison);
             WaitForGraphs();
 
-            PauseForPeakAreaGraphScreenShot("Peak Areas view");
+            PauseForScreenShot<GraphSummary.AreaGraphView>("Peak Areas View");
             return true;    // Continue subsequent tests
         }
 
         protected void DoResultsGridView()
         {
             // Results Grid View
             RestoreViewOnScreen(26);
+            RunUI(() =>
+            {
+                var areaGraphFloatingWindow = FindFloatingWindow(SkylineWindow.GraphPeakArea);

```

Comment:
I wonder if these values should be relative to the bottom-right of the Skyline window. That is how their placement looked to me in the screenshot. Consider, if we wanted to make the Skyline window a little bigger for some reason, what edges would we want this window placement anchored to?

Reply:
Yes, will fix.